src/config/common.js

```diff
@@ -48,6 +49,16 @@ export function getResolve(config, paths) {
         '.web.js', '.web.jsx', '.web.ts', '.web.tsx',
         '.js', '.json', '.jsx', '.ts', '.tsx',
       ],
+      alias: {
```

Comment:
Well. the default alias can be removed. In fact, I just want to have a way to define self alias.